UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Helpers/FssApiClientTests.cs

```diff
@@ -197,6 +197,28 @@ public void DoesDownloadFile_Returns_OK()
             });
         }
 
+        [Test]
+        public void DoesDownloadFile_With_Client_That_Allow_Redirect_As_False_Returns_OK()
+        {
+            string Content = "";
+
+            HttpMessageHandler? messageHandler = FakeHttpMessageHandler.GetHttpMessageHandler(
```

Comment:
This doesn't need to be nullable (same for all the similar methods in this class).